pkg/ddl/backfilling_dist_scheduler.go

```diff
@@ -426,14 +426,20 @@ func CalculateRegionBatch(totalRegionCnt int, nodeCnt int, useLocalDisk bool) in
 		failpoint.Return(val.(int))
 	})
 	var regionBatch int
+	if useLocalDisk {
+		// We want to avoid too may partial imports when using local disk. So we limit the node count to 3 so that
+		// at most 3 partial imports if disk space is enough.
```

Comment:
```suggestion
		// We want to avoid too may partial imports when using local disk. So we
		// limit the node count to 3 so that at most 3 partial imports if disk
		// space is enough.
```